lib/logic.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class TicTacLogic
+  attr_accessor :player1, :player2
```

Comment:
`attr_accessor` is not allowed, you should use either `attr_reader` or `attr_writer`